src/App/Model/CryptAES.php

```diff
@@ -37,7 +37,7 @@ public function decrypt($crypt)
     {
         if (!$this->cipher && !$this->setCipher()) return $crypt;
 
-        return $this->cipher->decrypt($crypt);
+        return $this->cipher->decrypt(base64_decode($crypt));
```

Comment:
The tests in phpseclib are using bin2hex/hex2bin:
https://github.com/phpseclib/phpseclib/blob/2.0/tests/Unit/Crypt/AES/TestCase.php